src/relay.js

```diff
@@ -356,7 +356,8 @@ export function sequelizeConnection({
     return {
       source,
       args,
-      where: argsToWhere(args)
+      where: argsToWhere(args),
+      context
```

Comment:
Changes in context are not observed for non edge fields, I think it keep using old context from higher levels somehow. I will try to add a test to demonstrate this.